src/cata_tiles.cpp

```diff
@@ -2757,6 +2758,21 @@ bool cata_tiles::draw_zone_mark( const tripoint &p, lit_level ll, int &height_3d
     return false;
 }
 
+bool cata_tiles::draw_zombie_revival_indicators( const tripoint &pos, const lit_level /*ll*/,
+        int &/*height_3d*/, const bool ( &invisible )[5] )
+{
+    if( tileset_ptr->find_tile_type( ZOMBIE_REVIVAL_INDICATOR ) && !invisible[0] &&
+        item_override.find( pos ) != item_override.end() && g->m.could_see_items( pos, g->u ) ) {
```

Comment:
```suggestion
        item_override.find( pos ) == item_override.end() && g->m.could_see_items( pos, g->u ) ) {
```
`==` means there's no override from the debug menu, so you need this instead.

Reply:
Whoops, right.